db/migrate/20150513034751_add_repository_branches.rb

```diff
@@ -0,0 +1,26 @@
+class AddRepositoryBranches < ActiveRecord::Migration
+  def up
+    create_table :katello_repository_branches do |t|
+      t.string :branch, :null => false
```

Comment:
https://github.com/projectatomic/rpm-ostree/blob/master/doc/treefile-examples/fedora-rawhide-base.json#L2 .. Ok more I see examples the more I am convinced it should be refs and not branch or path attribute name. Going to rename....